src/batch-transaction.ts

```diff
@@ -129,7 +188,7 @@ class BatchTransaction extends Transaction {
           reqOpts,
           gaxOpts,
         },
-        callback);
+        callback!);
   }
   /**
    * Generic create partition method. Handles common parameters used in both
```

Comment:
194. used in both what?

Reply:
For BatchTransaction#createQueryPartitions and BatchTransaction#createReadPartitions as I understood.